pillar/heroku/xpro.sls

```diff
@@ -37,7 +42,16 @@ heroku:
     AWS_ACCESS_KEY_ID:  __vault__:cache:aws-mitx/creds/read-write-xpro-app-{{ env_data.env_name }}>data>access_key
     AWS_SECRET_ACCESS_KEY: __vault__:cache:aws-mitx/creds/read-write-xpro-app-{{ env_data.env_name }}>data>secret_key
     AWS_STORAGE_BUCKET_NAME: 'xpro-app-{{ env_data.env_name }}'
+    CYBERSOURCE_ACCESS_KEY: {{ cybersource_creds.data.access_key }}
+    CYBERSOURCE_PROFILE_ID: {{ cybersource_creds.data.profile_id }}
+    CYBERSOURCE_REFERENCE_PREFIX: xpro-{{ env_data.env_name }}
+    CYBERSOURCE_SECURE_ACCEPTANCE_URL: {{ env_data.CYBERSOURCE_SECURE_ACCEPTANCE_URL}}
+    CYBERSOURCE_SECURITY_KEY: {{ cybersource_creds.data.security_key }}
+    {% if 'production' in env_data %}
+    DATABASE_URL: postgres://{{ pg_creds.data.username }}:{{ pg_creds.data.password }}@{{ rds_endpoint }}
```

Comment:
You'll need to append the database name to the end of the connection URI (e.g. `/xpro-production`)